src/routes/+layout.svelte

```diff
@@ -28,6 +45,7 @@
 			</a>
 		</svelte:fragment>
 		<div slot="actions" class="flex items-center justify-end w-full">
+            <Button variant="fill" on:click={handleTrigger} style="margin-right: 20px">Trigger Pipeline</Button>
```

Comment:
Same as the User button
```suggestion
            <Button variant="fill" on:click={handleTrigger} class="mr-4">Trigger Pipeline</Button>
```